src/index.html

```diff
@@ -8,6 +8,149 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div 
+      class="stars--0 stars"
+    >
+      <div
+        class="stars__star"
+      >
```

Comment:
if you have one short attribute it's better to write it in one line, fix in all places
```suggestion
      <div class="stars__star">
```